airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -142,3 +147,49 @@ def patch_dag_run(
     dag_run = session.get(DagRun, dag_run.id)
 
     return DAGRunResponse.model_validate(dag_run, from_attributes=True)
+
+
+@dag_run_router.post("/{dag_run_id}/clear", responses=create_openapi_http_exception_doc([401, 403, 404]))
```

Comment:
```suggestion
@dag_run_router.post("/{dag_run_id}/clear", responses=create_openapi_http_exception_doc([404]))
```

Convention changed here https://github.com/apache/airflow/pull/43932. 

401, 403 are now directly inherited from the base router.

Reply:
Done